types/sandbox/prerequest.d.ts

```diff
@@ -89,32 +103,59 @@ declare interface Visualizer {
     clear(): void;
 }
 
+declare interface Execution {
+    /**
+     * Halts the execution of current request. No line after this will be executed and
+     * if invoked from a pre-request script, the request will not be sent.
```

Comment:
This is prerequest.d.ts file, so you can write the doc assuming "if executed from pre-request script" to be true.
```suggestion
     * the request will not be sent.
```

Reply:
It is not separated like that only some part of jsdocs around response are omitted in case of pre-request script